doc/themes/the-things-stack/assets/js/theme.js

```diff
@@ -12,153 +12,155 @@
       if (item && item.classList.contains(activeClass)) {
-        item.classList.remove(activeClass)
+        item.classList.remove(activeClass);
       }
-      let data = item.getAttribute('data-content')
+      let data = item.getAttribute("data-content");
       if (data === key) {
-        item.classList.add(activeClass)
+        item.classList.add(activeClass);
       }
-    })
+    });
     if (window.sessionStorage) {
-      window.sessionStorage.setItem('tabActive', key)
+      window.sessionStorage.setItem("tabActive", key);
     }
 
-    return true
+    return true;
   }
 
-  const tabs = document.querySelectorAll('.tabs li')
-  const tabContent = document.querySelectorAll('.tab-content section')
-
-  function addTabs(){
+  const tabs = document.querySelectorAll(".tabs li");
+  const tabContent = document.querySelectorAll(".tab-content section");
 
-    if(tabs.length===0) return
+  function addTabs() {
+    if (tabs.length === 0) return;
 
     function handleClick(event) {
-      event.preventDefault()
-      let tab = event.currentTarget
-      let key = tab.getAttribute('data-tab')
-      toggleTab(key)
+      event.preventDefault();
+      let tab = event.currentTarget;
+      let key = tab.getAttribute("data-tab");
+      toggleTab(key);
     }
 
-    if (window.sessionStorage.getItem('tabActive')) {
-      const success = toggleTab(window.sessionStorage.getItem('tabActive'))
+    if (window.sessionStorage.getItem("tabActive")) {
+      const success = toggleTab(window.sessionStorage.getItem("tabActive"));
       if (success === false) {
-        let key = tabs[0].getAttribute('data-tab')
-        toggleTab(key)
+        let key = tabs[0].getAttribute("data-tab");
+        toggleTab(key);
       }
     } else if (tabs && tabs[0]) {
-      let key = tabs[0].getAttribute('data-tab')
-      toggleTab(key)
+      let key = tabs[0].getAttribute("data-tab");
+      toggleTab(key);
     }
 
-    tabs.forEach(tab => {
-      tab.addEventListener('click', handleClick)
-    })
-
+    tabs.forEach((tab) => {
+      tab.addEventListener("click", handleClick);
+    });
   }
 
-  addAnchors()
-  addTabs()
-  addOnHashChange()
-
+  addAnchors();
+  addTabs();
+  addOnHashChange();
 }
 
 function addLightboxHandler() {
-  document.querySelectorAll('figure').forEach(fig => {
-    fig.addEventListener('click', event => {
-      basicLightbox.create(`<img src="${event.target.src}" alt="${event.target.alt}" />`).show()
-    })
-  })
+  document.querySelectorAll("figure").forEach((fig) => {
+    fig.addEventListener("click", (event) => {
+      basicLightbox.create(`<img src="${event.target.src}" alt="${event.target.alt}" />`).show();
+    });
+  });
 }
 
-document.addEventListener('DOMContentLoaded', function () {
-  addNavBarBurgers()
-  initializeTabsAndAnchors()
-  addLightboxHandler()
-})
+document.addEventListener("DOMContentLoaded", function () {
+  addNavBarBurgers();
+  initializeTabsAndAnchors();
+  addLightboxHandler();
+});
```

Comment:
@pierrephz What is auto editing the js files here. It does not seem inline with the project.

Could you make sure you have https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig installed in vscode. 

Otherwise lets have a look at your vscode settings to check what default config it is using to edit on save

Reply:
fixed